src/vaults/VaultManager.ts

```diff
@@ -36,6 +36,7 @@ import * as vaultsEvents from './events';
 import * as utils from '../utils';
 import * as vaultsUtils from '../vaults/utils';
 import * as vaultsErrors from '../vaults/errors';
+import * as vaultsEvents from './events';
```

Comment:
This should be `./utils` and `./errors`.

Reply:
Fixed.